### ISSUE TITLE
Enable status LED

### ISSUE THREAD

**Issue by Celger** [Mange/rtl8192eu-linux-driver#241]
Please, considerer enable the LEDs by uncommenting lines in autoconf.h as recommended  in https://github.com/Mange/rtl8192eu-linux-driver/issues/138#issuecomment-705308182 after you closed the issue. Or make an option for that.
Thanks! You are doing a great work!

**Comment by CGarces** [comment]
I don't have this the device, so I can't test if the solution is valid.
Have you compiled and tested the proposed solution?

**Comment by Celger** [comment]
Yeah! I have TL-WN822N(EU) ver.5.0. I uncommented the LED lines and compiled. It worked like a charm!

Uncomment the lines 152, 154 and 156 in autoconf.h

**Comment by circl-lastname** [comment]
IS it necessary to uncomment the lines 154 and 156 rather than just 152?

**Comment by Celger** [comment]
According [https://github.com/Mange/rtl8192eu-linux-driver/issues/138#issuecomment-705308182](url) you're right. Uncomment the line 152 is all you need. But there is several references to these definitions, listed below, so I uncommented these three lines and, it's ok.

Files and lines where the definition in line 154 is used:
`hal/led/hal_usb_led.c:18:#ifdef CONFIG_RTW_SW_LED
hal/led/hal_led.c:27:#ifdef CONFIG_RTW_SW_LED
hal/led/hal_led.c:40:#ifndef CONFIG_RTW_SW_LED
hal/led/hal_led.c:42:		RTW_WARN("CONFIG_RTW_SW_LED is not defined\n");
hal/led/hal_led.c:47:#ifdef CONFIG_RTW_SW_LED
hal/led/hal_led.c:62:#ifdef CONFIG_RTW_SW_LED
hal/led/hal_led.c:69:#ifdef CONFIG_RTW_SW_LED
hal/led/hal_led.c:251:#endif /* CONFIG_RTW_SW_LED */
hal/rtl8192e/usb/usb_halinit.c:1445:#ifdef CONFIG_RTW_SW_LED
hal/rtl8192e/usb/usb_halinit.c:1562:#ifdef CONFIG_RTW_SW_LED
hal/rtl8192e/usb/usb_halinit.c:1573:#endif /* CONFIG_RTW_SW_LED */
hal/rtl8192e/usb/usb_halinit.c:1880:#ifdef CONFIG_RTW_SW_LED
hal/rtl8192e/usb/usb_halinit.c:1883:#endif/* CONFIG_RTW_SW_LED */
hal/rtl8192e/usb/rtl8192eu_led.c:19:#ifdef CONFIG_RTW_SW_LED
hal/hal_intf.c:202:#ifdef CONFIG_RTW_SW_LED
include/rtl8723b_led.h:22:#ifdef CONFIG_RTW_SW_LED
include/rtl8723b_led.h:44:#endif/*CONFIG_RTW_SW_LED*/
include/rtl8188f_led.h:17:#ifdef CONFIG_RTW_SW_LED
include/rtl8188f_led.h:45:#endif/*CONFIG_RTW_SW_LED*/
include/rtl8192f_led.h:22:#ifdef CONFIG_RTW_SW_LED
include/rtl8192f_led.h:40:#endif /*#ifdef CONFIG_RTW_SW_LED*/
include/rtl8723d_led.h:22:#ifdef CONFIG_RTW_SW_LED
include/rtl8723d_led.h:43:#endif /*#ifdef CONFIG_RTW_SW_LED*/
include/hal_intf.h:336:#ifdef CONFIG_RTW_SW_LED
include/hal_intf.h:660:#ifdef CONFIG_RTW_SW_LED
include/rtl8710b_led.h:22:#ifdef CONFIG_RTW_SW_LED
include/rtl8710b_led.h:43:#endif /*#ifdef CONFIG_RTW_SW_LED*/
include/autoconf.h:154:	#define CONFIG_RTW_SW_LED
include/autoconf.h:155:	#ifdef CONFIG_RTW_SW_LED
include/rtl8812a_led.h:18:#ifdef CONFIG_RTW_SW_LED
include/rtl8812a_led.h:34:#endif/*CONFIG_RTW_SW_LED*/
include/rtl8192e_led.h:18:#ifdef CONFIG_RTW_SW_LED
include/rtl8192e_led.h:36:#endif/*CONFIG_RTW_SW_LED*/
include/hal_com_led.h:276:#ifdef CONFIG_RTW_SW_LED
include/hal_com_led.h:349:#ifdef CONFIG_RTW_SW_LED
include/hal_com_led.h:407:#if defined(CONFIG_RTW_LED) && defined(CONFIG_RTW_SW_LED)
include/hal_com_led.h:409:#ifndef CONFIG_RTW_SW_LED_TRX_DA_CLASSIFY
include/hal_com_led.h:410:#define CONFIG_RTW_SW_LED_TRX_DA_CLASSIFY 0
include/hal_com_led.h:413:#if CONFIG_RTW_SW_LED_TRX_DA_CLASSIFY
include/hal_com_led.h:434:#endif /* defined(CONFIG_RTW_LED) && defined(CONFIG_RTW_SW_LED) */
include/rtl8703b_led.h:22:#ifdef CONFIG_RTW_SW_LED
include/rtl8703b_led.h:43:#endif/*CONFIG_RTW_SW_LED*/
include/rtl8814a_led.h:18:#ifdef CONFIG_RTW_SW_LED
include/rtl8814a_led.h:36:#endif /*CONFIG_RTW_SW_LED*/
include/rtl8188e_led.h:18:#ifdef CONFIG_RTW_SW_LED
include/rtl8188e_led.h:37:#endif /*CONFIG_RTW_SW_LED*/
os_dep/linux/os_intfs.c:2448:#ifdef CONFIG_RTW_SW_LED
os_dep/linux/os_intfs.c:2518:#ifdef CONFIG_RTW_SW_LED
os_dep/linux/os_intfs.c:4873:#ifdef CONFIG_RTW_SW_LED
os_dep/linux/os_intfs.c:5008:#ifdef CONFIG_RTW_SW_LED
`
